pkg/controllers/node/termination/suite_test.go

```diff
@@ -809,8 +808,6 @@ var _ = Describe("Termination", func() {
 			node = ExpectNodeExists(ctx, env.Client, node.Name)
 			ExpectObjectReconciled(ctx, env.Client, terminationController, node)
 			ExpectSingletonReconciled(ctx, queue) // Reconcile the queue so that we set the metric
-
-			ExpectMetricGaugeValue(terminator.EvictionQueueDepth, 5, map[string]string{})
```

Comment:
Is there a way for us to check the workqueue metric to get this eviction queue depth?